### PR TITLE
Add bordered UI feature and slot support

### LINKED ISSUE #53: Develop new outline element modifier
## Is your feature request related to a problem? Please describe.

The current UI for auro-input has caused some issues and created design limitations. 

## Describe the solution you'd like

Add an additional style that uses a full outline for the auro-input custom element. 

- https://github.com/AlaskaAirlines/auro-input/issues/52

Figma comp:

https://www.figma.com/file/ClsrU5j9yHiLA35bq7IRuO/Input?node-id=401%3A287

### PR DESCRIPTION
**Pull request by blackfalcon**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #53, #59, #69

## Demo

https://auro-input-issue53.surge.sh/

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

This pull request resolves #53 by applying a new UI variant option for outlined input elements. 

### Screen shot iOS

Screen Shot 2021-10-23 at 11 07 52 AM

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Review by jason-capsule42 — DISMISSED** [review]

**Comment by leeejune** [comment]
Made edits to the blueprint to add `Date` type.

**Comment by blackfalcon** [comment]
Screen Shot 2021-11-01 at 12 31 05 PM

@leeejune the default text should be,
```Please enter a valid credit card number.```

The error message will be the same as well.

**Comment by blackfalcon** [comment]
@leeejune Please set up a time where we can review this. I need some help understanding what the issue is with these two? Updates posted to http://auro-input-issue53.surge.sh/

Screen Shot 2021-11-01 at 12 34 44 PM

**Comment by leeejune** [comment]
@blackfalcon The issue was that the content height was different. For example, the label should have a height of 16px, but it has a height of 24px (32px with padding in screenshot).

**Comment by leeejune** [comment]
> Screen Shot 2021-11-01 at 12 31 05 PM
> 
> @leeejune the default text should be, `Please enter a valid credit card number.`
> 
> The error message will be the same as well.

Then when should the `Valid credit card numbers must include 16 digits (15 for Amex).` be displayed? I referred to this issue: https://github.com/AlaskaAirlines/auro-input/issues/69#issuecomment-952094248

**Review by leeejune — APPROVED** [review]

File: .stylelintrc [hunk 1]
```diff
@@ -100,7 +100,7 @@
     "selector-max-combinators": 1,
     "selector-max-compound-selectors": 2,
     "selector-max-id": 1,
-    "selector-max-pseudo-class": 1,
+    "selector-max-pseudo-class": 2,
```
[the remark this answers is not in the document]
Comment (blackfalcon) [reply .stylelintrc:103]: Discussion moved to https://github.com/AlaskaAirlines/WC-Generator/issues/246

**Pull request merged**